scripts/sct_propseg.py

```diff
@@ -509,6 +517,28 @@ def generate_qc(fn_in, fn_seg, args, path_qc):
     # enabling centerline extraction by default (needed by check_and_correct_segmentation() )
     cmd += ['-centerline-binary']
 
+    # rescale header (see issue #1406)
+    rescale_header = 2  # TODO add flag
+    if rescale_header:
+        # TODO: rescale header of image and init-centerline/mask
+        import nibabel as nib
+        img = nib.load(fname_data)
+        # get qform
+        qform = img.header.get_qform()
+        # multiply by scaling factor
+        qform[0:3, 0:3] *= rescale_header
+        # generate a new nifti file
+        header_rescaled = img.header.copy()
+        header_rescaled.set_qform(qform)
+        # the data are the same-- only the header changes
+        img_rescaled = nib.nifti1.Nifti1Image(img.get_data(), None, header=header_rescaled)
+        path_tmp = sct.tmp_create(basename="propseg", verbose=verbose)
```

Comment:
Looks like the temp. folder is not removed at the end.

Reply:
indeed-- i've added a todo (not urgent)